TwilioVerifyDemo/TwilioVerifyDemo/Common/KeyChainAccessGroupHelper.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  KeyChainAccessGroupHelper.swift
```

Comment:
- Should this file have the `Licensed under the Apache License` copyright header? 
- What if you set all this helper + the structure internal? I think since is part of the Demo app it make more sense